.github/workflows/transports-release.yml

```
@@ -106,17 +106,21 @@ jobs:
       # Determine versions and create transport tag
       - name: Create transport tag
         id: manage_versions
-        working-directory: .
         run: |
           # Get current core version from go.mod and generate new transport version
           node ci/scripts/manage-versions.mjs transport-release >> "$GITHUB_OUTPUT"
           
-          # Get the transport version for tagging
+          # Store the transport version in a shell variable
           TRANSPORT_VERSION=$(grep "transport_version=" "$GITHUB_OUTPUT" | cut -d'=' -f2)
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)